.github/workflows/policy_controller.yml

```
@@ -24,6 +24,8 @@ env:
   PROXY_INIT_VERSION: v1.5.3
   RUST_BACKTRACE: short
   RUSTUP_MAX_RETRIES: 10
+  PROTOC_NO_VENDOR: 1
+
```

could we just set the `PROTOC` environment variable here, so it applies to all actions
```suggestion
  PROTOC: "$PWD/bin/protoc"
```
or would `$PWD` not get interpolated correctly?

---

Right, we can't reference other variables in this section.